vocabularies/geoadmin-feature-types.ttl

```diff
@@ -0,0 +1,69 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix geof: <http://linked.data.gov.au/def/geofeatures/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix prof: <http://www.w3.org/ns/dx/prof/> .
```

Comment:
The prof: prefix is not used in this vocab

Reply:
As above: legacy of copy 'n paste from previous script. Removed.